roles/pulp_devel/tasks/rpm_signing_service.yml

```diff
@@ -0,0 +1,63 @@
+---
+# Setup the RPM signing service
+#
+# Maintain parity with:
+# https://github.com/pulp/pulp_rpm/blob/master/.travis/post_before_script.sh
+
+- name: Copy the signing service script to {{ __pulp_devel_rpm_sign_script_path }}
+  copy:
+    src: >-
+      {{ pulp_install_plugins_normalized['pulp-rpm'].source_dir | default (developer_user_home ~ '/devel/pulp_rpm')
```

Comment:
These 3 syntax examples will be very useful in the future:

- Eliminating the outer quotes with a black
- The use of default() with a variable
- The concatenation of the variable with the string

Reply:
However, this file will only be executed, if `pulp_install_plugins_normalized['pulp-rpm'].source_dir` is defined...